projects/ngx-diagrams/src/lib/models/move-selection.model.ts

```diff
@@ -1,9 +1,8 @@
 import { BaseModel } from './base.model';
-import { BaseEntity } from '../base.entity';
 import { PortModel } from './port.model';
 
 export interface SelectionModel {
-	model: BaseModel<BaseEntity>;
+	model: BaseModel;
```

Comment:
Why would you want to remove the BaseEntity Generics? all entities in SelectionModel should inherit from BaseEntity to avoid breaking the API

Reply:
This one specifically seems like a lint auto fix - but as I see, `BaseModel` uses `BaseEntity` as default generic anyway

`export class BaseModel<X extends BaseEntity = BaseEntity> extends BaseEntity`